person.rb

```diff
@@ -1,13 +1,8 @@
-require_relative 'nameable'
-class Person < Nameable
-  attr_reader :name
+class Person
+  attr_accesor :name, :rentals
 
   def initialize(name)
-    super
     @name = name
-  end
-
-  def correct_name
-    @name
+    @rentals = []
   end
 end
```

Comment:
- [ ] For these associations to work we need to be able to create a Person in the first place. Kindly go back to your previous code ([visit this file in this PR](https://github.com/Ibtisam34/OOP-school-library/pull/1/files#diff-47720676886debe217f2360ee91e7dcbfca82989297cf253a11792456731033f) and get your code back) and re-add the missing lines of code that allow us to create a Person.

  Also, bring back all the missing methods this class once had in earlier reviews. We are building on previous code, there is no need to delete this amount of code.

- [ ] Be sure to maintain the `@rentals` instance variable in the `initialize` method.

  ```
  def initialize(...)
    ...
    @rentals = [ ]
  end
  ```

- [ ] Kindly add the method for establishing an association with the Book and Rental classes.

  ```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  ```

Reply:
- Well resolved @Ibtisam34